### ISSUE TITLE
NVDA is Announcing "Replace" Button Thrice after Activating "Next" Button.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14003]
### Steps to reproduce:
Step 1: Open  the URL given below in edge dev and login with valid credentials.
Step 2: Navigate to "Billing" and expand it.
Step 3: Navigate to "Payment methods" and activate it.
Step 4: Navigate to "Replace" tab and activate it.
Step 5: Select "Use another Payment method" radio button.
Step 6: Turn On NVDA
Step 7: Activate Next button and observe NVDA announcement.

### Actual behavior:

NVDA is announcing "Replace" button thrice after activating "Next" button. It is announcing "Replace Replace Replace review Microsoft 365 admin center".

### Expected behavior:

NVDA should announce "Replace" button only once. It should announce like "Replace review Microsoft 365 admin center".

#### NVDA version:
2022.1
#### Windows version:

Windows 11 version 22H2(OS Build 22621.1)

#### Name and version of other software in use when reproducing the issue:
Edge version(Version 104.0.1293.47 (Official build) (64-bit))
URL- https://admin-ignite.microsoft.com/adminportal/home?pkgver=PR_220607.101_V2#/PaymentMethods
![download](https://user-images.githubusercontent.com/93735775/183744112-ca731f08-1052-48d8-ac1a-9e5ed74fb3ab.jpg)

https://user-images.githubusercontent.com/93735775/183743096-6f99ff26-93b2-4421-927a-0989b19ad37d.mp4

#### Does the issue still occur after restarting your computer?
Yes

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers
